app/services/check_breaks_in_work_history.rb

```diff
@@ -3,34 +3,33 @@ class << self
     def call(application_form)
       jobs = application_form.application_work_experiences.sort_by(&:start_date)
 
-      return false if jobs.empty?
-      return break_between_only_job_and_current_date?(jobs.first) if jobs.one?
+      previous_end_date = nil
```

Comment:
```suggestion
      latest_end_date = nil
```

Is this a more intuitive name? Minor